docs/release-notes/certificates-import.rst

```diff
@@ -4,7 +4,8 @@
 :Description:
     New kickstart section %certificate is supported.
     It allows users to securely embed certificates directly within
-    the kickstart file.
+    the kickstart file. The certificates are imported both
+    to the installer environment and the installed system.
```

Comment:
I think it's 'import into' to 'import to' 
https://forum.wordreference.com/threads/import-files-into-in-to-a-folder.3089279/